RELEASE_NOTES.md

```diff
@@ -1,5 +1,13 @@
 _This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes._
 
+# Chef Client Release Notes 13.9.1:
+
+## Platform Additions
+
+- Enable Ubuntu-18.04 tested chef-client packages.
```

Comment:
Could we just simplify this to say "We now build and test chef-client for both Ubuntu 18.04 and Debian 9". Seems like we're getting into the weeds a bit mentioning the builders.

Reply:
👍 done